sys/cpp11-compat/thread.cpp

```diff
@@ -42,7 +42,7 @@ void thread::join() {
   }
   if (joinable()) {
     auto status = thread_getstatus(m_handle);
-    if (status != (int)STATUS_NOT_FOUND && status != STATUS_STOPPED) {
+    if (status != STATUS_NOT_FOUND && status != STATUS_STOPPED) {
```

Comment:
better add some parenthesis here to make the evaluation order clear.

Reply:
What do you mean? The order should not matter. 
`status != -1 && status != 0`

Reply:
Oh, nvm I thought it was `&` instead of `&&`.